File: ekka-tcp/src/main/java/ekkatcp/server/MessageType.java
package ekkatcp.server;

public interface MessageType {
    static byte SIGNIN_REQ = 1;
    static byte SIGNIN_RES = 2;
    static byte PING = 3;
    static byte PONG = 4;
    static byte SEND_MSG = 5;
    static byte SEND_MSG_ACK = 6;
    static byte NOTI_MSG = 7;
    static byte NOTI_MSG_ACK = 8;
}

File: ekka-srv/src/main/scala/ekka/srv/MessageType.scala
package ekka.srv

final object MessageType {
  val SIGNIN_REQ: Byte = 1
  val SIGNIN_RES: Byte = 2
  val PING: Byte = 3
  val PONG: Byte = 4
  val SEND_MSG: Byte = 5
  val SEND_MSG_ACK: Byte = 6
  val NOTI_MSG: Byte = 7
  val NOTI_MSG_ACK: Byte = 8
}
